iquip/apps/logger.py

```diff
@@ -125,52 +137,75 @@ class LoggerApp(qiwis.BaseApp):
     Attributes:
         loggerFrame: A frame that shows the logs.
         confirmFrame: A frame that asks whether to clear logs.
-        handler: A handler for adding logs to the loggerFrame. 
+        levelsDict: A dictionary that matches int level to string level.
+        frameHandler: A handler for adding logs to the loggerFrame.
+        fileHandler: A handler for saving logs to file.
     """
 
-    def __init__(self, name: str, parent: Optional[QObject] = None):
-        """Extended."""
+    def __init__(self, name: str, path: str, parent: Optional[QObject] = None):
+        """Extended.
+        
+        Args:
+            name: Specifies name of the LoggerApp.
+            path: Desribes file directory of log record file.
```

Comment:
How about "The path of log recodrd file".
I recommend to write it in noun form.